.packit.yaml

```diff
@@ -36,14 +36,26 @@ jobs:
     get-current-version:
     - grep -oP '^Version:\s+\K\S+' packaging/convert2rhel.spec
 - &tests
-  job: tests
-  targets:
-    epel-7-x86_64:
-      distros: [centos-7, oraclelinux-7]
-    epel-8-x86_64:
-      distros: [centos-8.4, centos-8-latest, oraclelinux-8.4, oraclelinux-8.6]
-  use_internal_tf: True
-  trigger: pull_request
+  - job: tests
```

Comment:
the extra 2 spaces make these nest under a job and not in the `jobs` section, hence making our parser go bananas

Reply:
Oh, thank you, @TomasTomecek !
What would be the best way to approach this then?